pkg/controllers/jobset_controller.go

```diff
@@ -669,16 +669,16 @@ func labelAndAnnotateObject(obj metav1.Object, js *jobset.JobSet, rjob *jobset.R
 	labels := util.CloneMap(obj.GetLabels())
 	labels[jobset.JobSetNameKey] = js.Name
 	labels[jobset.ReplicatedJobNameKey] = rjob.Name
-	labels[RestartsKey] = strconv.Itoa(js.Status.Restarts)
-	labels[jobset.ReplicatedJobReplicas] = strconv.Itoa(rjob.Replicas)
-	labels[jobset.JobIndexKey] = strconv.Itoa(jobIdx)
+	labels[RestartsKey] = strconv.Itoa(int(js.Status.Restarts))
+	labels[jobset.ReplicatedJobReplicas] = strconv.Itoa(int(rjob.Replicas))
+	labels[jobset.JobIndexKey] = strconv.Itoa(int(jobIdx))
 	labels[jobset.JobKey] = jobHashKey(js.Namespace, jobName)
 
 	annotations := util.CloneMap(obj.GetAnnotations())
 	annotations[jobset.JobSetNameKey] = js.Name
 	annotations[jobset.ReplicatedJobNameKey] = rjob.Name
-	annotations[jobset.ReplicatedJobReplicas] = strconv.Itoa(rjob.Replicas)
-	annotations[jobset.JobIndexKey] = strconv.Itoa(jobIdx)
+	annotations[jobset.ReplicatedJobReplicas] = strconv.Itoa(int(rjob.Replicas))
+	annotations[jobset.JobIndexKey] = strconv.Itoa(int(jobIdx))
```

Comment:
We don't need to do `int(jobIdex)` anymore since it is now an int instead of int32.

Reply:
nice suggestion! Fixed!